atlascope/core/rest/endpoints/dataset_endpoints.py

```diff
@@ -44,3 +50,21 @@ def create(self, request):
         )
 
         return Response(DatasetSerializer(new_dataset_obj).data, status=status.HTTP_201_CREATED)
+
+    @swagger_auto_schema(request_body=DatasetSubImageSerializer())
+    @action(detail=False, methods=['POST'])
+    def create_sub_image(self, request):
+        serializer = DatasetSubImageSerializer(data=self.request.data)
+        serializer.is_valid(raise_exception=True)
+        original_dataset = Dataset.objects.get(id=serializer.validated_data["original_dataset_id"])
+        meta = serializer.validated_data.copy()
+        meta.pop("original_dataset_id")
+        new_dataset_obj = Dataset(
+            name=f'{original_dataset.name} Sub Image',
```

Comment:
```suggestion
            name=f'{original_dataset.name} Subimage',
```

Reply:
In the future, we may find it useful to be able to override this name. No need to put in that feature now.